tap_google_analytics/client.py

```diff
@@ -125,9 +149,10 @@ def _make_request(self, method, url, params=None, data=None):
         else:
             response = self.session.request(method, url, headers=headers, params=params)
 
-        error_message = _is_json(response) and response.json().get("error", {}).get("message")
-        if response.status_code == 400 and error_message:
-            raise Exception("400 Client Error: Bad Request, details: {}".format(error_message))
+        response_is_json = _is_json(response)
+        error_message = response_is_json and response.json().get("error", {}).get("message")
+        if response.status_code in [400, 403] and error_message:
+            raise Exception("{} Error from Google - Details: {}".format(response.status_code, error_message))
```

Comment:
What version of python are required for the singer taps? Can we migrate to using f-strings instead of .format?

Looks like it's python 3 >= 3.6: https://realpython.com/python-f-strings/

Reply:
Our base is still 3.5.2, but yeah I'd like to use f-strings more for sure

Reply:
Alright if that's the case then leave the format approach for now